plugins/arRestApiPlugin/modules/api/actions/digitalobjectsCreateAction.class.php

```diff
@@ -54,7 +54,9 @@ protected function post($request, $payload)
         'format_name' => 'formatName',
         'format_version' => 'formatVersion',
         'format_registry_key' => 'formatRegistryKey',
-        'format_registry_name' => 'formatRegistryName'
+        'format_registry_name' => 'formatRegistryName',
+        'relative_path_within_aip' => 'relativePathWithinAip',
+        'aip_name' => 'aipName',
```

Comment:
Just curious if this trailing comma may be code guideline issue since I don't see one in the deleted line?

Reply:
I can remove this comma. 👍